addons/common/fnc_handleArsenalOpened.sqf

```diff
@@ -0,0 +1,66 @@
+#define DEBUG_MODE_FULL
+#include "script_component.hpp"
+
+#define IDC_RSCDISPLAYARSENAL_LIST            960
+#define IDC_RSCDISPLAYARSENAL_TAB_CARGOMISC   24
+#define IDC_RSCDISPLAYARSENAL_SORT            800
```

Comment:
These are already part of:
```cpp
#include "\a3\ui_f\hpp\defineResinclDesign.inc"
```
But that file cannot be included in the components script_component.hpp, because it has a double define that crashes the binarizer. It can be included in this file though.